tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.js

```diff
@@ -0,0 +1,142 @@
+      page,
+      dashboardPage.advancedParametersLink,
+      dashboardPage.importLink,
+    );
+
+    await importPage.closeSfToolBar(page);
+
+    const pageTitle = await importPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(importPage.pageTitle);
+  });
+
+  it(`should download ${sampleFile.args.type} sample file`, async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'downloadFile', baseContext);
+
+    sampleFilePath = await importPage.downloadSampleFile(page, sampleFile.args.type);
+
+    const doesFileExist = await files.doesFileExist(sampleFilePath);
+    await expect(doesFileExist, `${sampleFile.args.type} sample file was not downloaded`).to.be.true;
+  });
+
+  describe('Import file', async () => {
+    it(`should upload ${sampleFile.args.type} sample text file`, async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'importFile', baseContext);
+
+      await files.renameFile(sampleFilePath, newPath);
+
+      const uploadSuccessText = await importPage.uploadSampleFile(page, sampleFile.args.dropdownValue, newPath);
+      await expect(uploadSuccessText).contain(sampleFile.args.type);
+    });
+
+    it('should go to next import file step', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'nextStep', baseContext);
+
+      const panelTitle = await importPage.goToImportNextStep(page);
+      await expect(panelTitle).contain(importPanelTitle);
+    });
+
+    it('should start import file', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'confirmImport', baseContext);
+
+      const modalTitle = await importPage.startFileImport(page);
+      await expect(modalTitle).contain(importModalTitle);
+    });
+
+    it('should close import progress modal', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'closeImportModal', baseContext);
+
+      const fileTypeDropdownSelector = await importPage.closeImportModal(page);
+      await expect(fileTypeDropdownSelector).to.be.true;
+    });
+  });
+
+  describe('Check import success', async () => {
+    it('should go to customers page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToCustomersPage', baseContext);
+
+      await importPage.goToSubMenu(
+        page,
+        importPage.customersParentLink,
+        importPage.customersLink,
+      );
+
+      await customersPage.closeSfToolBar(page);
+
+      const pageTitle = await customersPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(customersPage.pageTitle);
+    });
+
+    it('should check import success', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkImportSuccess', baseContext);
+
+      const customersEmailList = await customersPage.getAllRowsColumnContent(page, 'email');
+      await expect(customersEmailList).contain(emailToCheck);
```

Comment:
I think that `customersEmailList` should be equal `emailToCheck` because we import all customers list.